--- ../moon-orig/moonraker/components/update_manager.py	2021-05-06 08:59:34.123785353 -0400
+++ moonraker/components/update_manager.py	2021-05-06 08:28:24.309650293 -0400
@@ -15,6 +15,7 @@
 import asyncio
 import time
 import tempfile
+import site
 import tornado.gen
 from tornado.ioloop import IOLoop, PeriodicCallback
 from tornado.httpclient import AsyncHTTPClient
@@ -529,13 +530,16 @@
                 raise config.error("No match for 'env_package_path': %s"
                                    % (env_package_path,))
         for opt in ["repo_path", "env", "python_reqs", "install_script",
-                    "python_dist_path", "env_package_path"]:
+                    "env_package_path"]:
             val = getattr(self, opt)
             if val is None:
                 continue
             if not os.path.exists(val):
                 raise config.error("Invalid path for option '%s': %s"
                                    % (val, opt))
+        if not os.path.exists(getattr(self, "python_dist_path")):
+             setattr(self, "python_dist_path", site.getsitepackages()[0])
+
 
     def _get_version_info(self):
         ver_path = os.path.join(self.repo_path, "scripts/version.txt")
